README.md

```diff
@@ -4,25 +4,46 @@ Infrastructure as Code for the GC Forms environment.
 
 ## Running Lambdas and DBs locally
 
-You will need to have the following installed on a MacOS machine. To install these pre-requisits a quick google search using the names of these dependencies should yield many tutorials on how to do so
+You will need to have the following installed on a MacOS machine.
 
 Pre-requisites:
-- Docker Hub
-- LocalStack
-- Homebrew
-- Terragrunt
-- AWS CLI
+- Docker Hub: https://docs.docker.com/desktop/mac/install/
+
+- Homebrew:
+    ```bash
+     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+     ```
+- LocalStack : `brew install localstack`
+- Terragrunt: 
+
+  1. `brew install warrensbox/tap/tfswitch`
+  1. `tfswitch 1.0.10`
+  1. `brew install warrensbox/tap/tgswitch`
+  1. `tgswitch 0.35.6`
+
+- Yarn: `brew install yarn`
+
+- AWS CLI: `brew install awscli`
+
 - AWS SAM CLI
-- Postgres and PGAdmin
 
-### Installing AWS SAM CLI
+  Please run these commands to install the aws sam cli using homebrew. The AWS SAM CLI is what we use to run the lambda functions locally and invoke them.
+
+  1. `brew tap aws/tap`
+  1. `brew install aws-sam-cli`
+
+- Postgres and PGAdmin
+  1. `brew install postgresql`
+  1. `brew install --cask pgadmin4`
 
-Please run these commands to install the aws sam cli using homebrew. The AWS SAM CLI is what we use to run the lambda functions locally and invoke them.
+### Starting LocalStack and E2E testing from devcontainers
 
-Install AWS SAM-CLI
-`brew tap aws/tap`
-`brew install aws-sam-cli`
+1. forms-terraform: **in** the devcontainer, run `.devcontainer/scripts/terraform_apply_localstack.sh`
+1. forms-terraform: while **not** in the devcontainer, run `make lambdas`
+1. platform-forms-client: **in** the devcontainer, cd /migrations; run `yarn install` and `node index.js`
+1. platform-forms-client: **in** the devcontainer, run `yarn install` and `yarn dev`
```

Comment:
What would you think of adding script blocks?

1. forms-terraform: while **not** in the devcontainer:
```sh
.devcontainer/scripts/terraform_apply_localstack.sh
```

2. platform-forms-client: **in** the devcontainer:
```sh
make lambdas
```

3. platform-forms-client: **in** the devcontainer:
```sh
cd /migrations
yarn install
node index.js

cd ..
yarn install
yarn dev
```